code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2172,3 +2182,24 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		M.adjust_fire_stacks(1)
 		M.IgniteMob()
 	..()
+
+/datum/reagent/consumable/ethanol/painkiller
+	name = "Painkiller"
+	description = "Dulls your pain. Not sure if that's a good thing or a bad thing."
+	boozepwr = 20
+	color = "#EAD677"
+	quality = DRINK_NICE
+	taste_description = "sugary and tart"
+	glass_icon_state = "painkiller"
+	glass_name = "Painkiller"
+	glass_desc = "A combination of tropical juices and rum. Surely this will make you feel better."
+
+/datum/reagent/consumable/ethanol/painkiller/on_mob_life(mob/living/carbon/M)
+	M.hal_screwyhud = SCREWYHUD_HEALTHY
```

Comment:
maybe add a point or so per chem tick (2 seconds) of stamina regen too?

Reply:
No, this isn't meant to be a buff, just a quirky cocktail.